kmath-core/src/commonMain/kotlin/space/kscience/kmath/operations/BufferAlgebra.kt

```diff
@@ -35,13 +35,11 @@ public interface BufferAlgebra<T, out A : Algebra<T>> : Algebra<Buffer<T>> {
     public fun Buffer<T>.zip(other: Buffer<T>, block: A.(left: T, right: T) -> T): Buffer<T> =
         zipInline(this, other, block)
 
-    @UnstableKMathAPI
     override fun unaryOperationFunction(operation: String): (arg: Buffer<T>) -> Buffer<T> {
```

Comment:
Not sure about that. I still do not like the API much. I have a feeling that we will need to make a separate type for operations like it is done in TF for example.

Reply:
It is Kotlin 1.6 restriction: experimental annotations can only be applied if the base method is annotated.